CMakeLists.txt

```diff
@@ -108,9 +108,11 @@ function(importLibraries)
     "Linux:librpm"
     "Linux:libudev"
     "Linux,Darwin,Windows:libxml2"
+    "Darwin,Windows:lief"
     "Linux,Darwin,Windows:linenoise-ng"
     "Linux,Darwin:lldpd"
     "Linux,Darwin,Windows:lzma"
+    "Windows,Darwin:mbedtls"
```

Comment:
mbedtls is a TLS library. Is this a hard dependency, or can we use openssl? (And why not linux)

Reply:
Yes, mbedtls is a hard dependency for LIEF, i believe its used to extract certificate information. I dont think openssl can be used as a replacement?  Currently this PR does not support Linux/ELF parsing, if ELF parsing should be included then i can update this to include Linux. Otherwise it could be updated in an additional PR?

Reply:
If it's a dependency, then that's what it is...